tchannel/container/heap.py

```diff
@@ -0,0 +1,86 @@
+from __future__ import absolute_import
+
+
+class HeapOperation(object):
+    def cmp(self, i, j):
+        """Compare the items in position i and j of the heap.
+
+        :param i: the first item's position of the heap list
+        :param j: the second item's position of the heap list
+        :return true if items[i] < items[j], otherwise false.
+        """
```

Comment:
cmp implies that you're doing the [`cmp`](https://docs.python.org/2/library/functions.html#cmp) operation. Call this `lt` or actually use `cmp` (in which case `if h.cmp(a, b)` becomes `if h.cmp(a, b) < 0`).